models/nft/ethereum/nft_ethereum_wallet_pnl.sql

```diff
@@ -11,65 +11,98 @@
     )
 }}
 
-WITH trades as
+WITH 
+
+{% if is_incremental() %}
+
+weekly_unique_wallet_address as 
+(
+    SELECT DISTINCT
```

Comment:
am i crazy or are these two queries being union'ed exactly the same?

Reply:
haha no you're not ser. Just fixed, should have been **seller**